BookPlayer/Player/PlayerManager.swift

```diff
@@ -407,15 +407,20 @@ extension PlayerManager: AVAudioPlayerDelegate {
     func audioPlayerDidFinishPlaying(_ player: AVAudioPlayer, successfully flag: Bool) {
         guard flag else { return }
 
-        player.currentTime = player.duration
-
         UserDefaults.standard.removeObject(forKey: Constants.UserDefaults.lastPlayedBook.rawValue)
 
+        self.currentBook?.setCompletionState()
+
+        self.currentBook?.resetProgress()
+
+        self.pause()
```

Comment:
I don't think this `pause` is necessary here either, can't see the difference when I remove it